integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/DistroXClusterCreationTest.java

```diff
@@ -181,7 +181,7 @@ public void testDistroxCloudStorage(MockedTestContext testContext) {
                 .withImageSettings(DIX_IMG_KEY)
                 .withNetwork(DIX_NET_KEY)
                 .when(distroXClient.create(), key(DISTRO_X_STACK))
-                .await(STACK_AVAILABLE)
+                .await(STACK_AVAILABLE, key(DISTRO_X_STACK))
```

Comment:
what required this change?

Reply:
I am not sure it is necessary. I had trouble getting the integration tests to run, and I made this change because it seemed like a good idea at the time. I know that the tests are passing now, so at worst, it doesn't hurt anything.

I will revert this change and see if the tests still pass.

Reply:
Clarification - I'll try removing the change after merging this :)